languages/clojure/exercises/concept/strings/.meta/example.clj

```diff
@@ -0,0 +1,20 @@
+(ns strings
+  (:require [clojure.string :as str]))
+
+(defn message
+  "Takes a string representing a log line
+   and returns its message with whitespace trimmed."
+  [s]
+  (str/trim (last (str/split s #":"))))
```

Comment:
@paparomeo did we mention the `#` in "#:" somewhere? I am thinking if a new learner knows about this usage...

Reply:
Good point, this requires knowledge of regex. We'll have to introduce that first, unless maybe there's another way?

Reply:
Using regular expressions is indeed a bit problematic, as the docs state:

> The implementation should only use language features introduced by the exercise or its prerequisites

I would preferrably not expose students to regular expressions at this point :) Is there perhaps a different way of writing this? One that does rely on a regular expression?